Added 12 new Q&A pairs to the Support knowledgebase section (#1081)
wandb/docs: content/support/how_can_i_see_files_that_are_not_under_the_files_tab.md

---

> title: How can I see files that are not under the Files tab?

Reviewer: ```suggestion
title: How can I see files that do not appear in the Files tab?
```

toc_hide: true
type: docs
tags:
  - experiments
---

> The Files tab is limited to 10,000 files. Use the Public API to download hidden files:

Reviewer: ```suggestion
The Files tab shows a maximum of 10,000 files. To download all files, use the [public API]({{< relref "/ref/python/public-api/api.md" >}}):
```
